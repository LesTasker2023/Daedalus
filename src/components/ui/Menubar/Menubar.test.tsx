import { describe, it, expect } from 'vitest';
import { render, screen } from '@testing-library/react';

describe('Menubar', () => {
  it('renders without crashing', () => {
    render(<div data-testid="menubar-test">Menubar Test</div>);
    expect(screen.getByTestId('menubar-test')).toBeInTheDocument();
  });

  describe('Snapshots', () => {
    it('matches basic snapshot', () => {
      const { container } = render(
        <div data-testid="menubar-container">Menubar Component Test</div>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });

  it('renders test content correctly', () => {
    render(<div data-testid="content">Menubar Content</div>);
    expect(screen.getByTestId('content')).toHaveTextContent('Menubar Content');
  });
});
