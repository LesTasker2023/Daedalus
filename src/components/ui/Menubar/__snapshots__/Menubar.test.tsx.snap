// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Menubar > Snapshots > matches basic snapshot 1`] = `
<div
  data-testid="menubar-container"
>
  Menubar Component Test
</div>
`;
